CSC8503CoreClasses/Level.h

```diff
@@ -1,8 +1,34 @@
 #pragma once
+#include "Room.h"
+using namespace NCL::Maths;
+
 namespace NCL {
 	namespace CSC8503 {
+		class Vent;
+		class GameObject;
 		class Level {
-
+		public:
+			Level(int levelID);
+			~Level() {}
+			int GetGuardCount() const { return mGuardCount; }
+			int GetCCTVCount() const { return mCCTVCount; }
+			Vector3 GetPrisonPosition() const { return mPrisonPosition; }
+			Vector3 GetPlayerStartPosition(int player) { return mPlayerStartPositions[player]; }
+		protected:
+			int mLevelID;
+			std::map<Vector3, GameObject*> mTileMap;
+			std::map<Vector3, Room> mRoomList;
+			std::vector<std::vector<Vector3>> mGuardPaths;
+			int mGuardCount;
+			std::vector<Vector3> mCCTVPositions;
+			std::vector<Vector3> mCCTVRotations;
```

Comment:
Would it make sense to hold these as a transform i.e. a matrix4? If we're storing both position and rotation